go/vt/vttablet/onlineddl/schema.go

```diff
@@ -81,6 +81,8 @@ const (
 	alterSchemaMigrationsComponentThrottled            = "ALTER TABLE _vt.schema_migrations add column component_throttled tinytext NOT NULL"
 	alterSchemaMigrationsCancelledTimestamp            = "ALTER TABLE _vt.schema_migrations add column cancelled_timestamp timestamp NULL DEFAULT NULL"
 	alterSchemaMigrationsTablePostponeLaunch           = "ALTER TABLE _vt.schema_migrations add column postpone_launch tinyint unsigned NOT NULL DEFAULT 0"
+	alterSchemaMigrationsStage                         = "ALTER TABLE _vt.schema_migrations add column stage text not null"
```

Comment:
text seems a little excessive, no? In case we want to use this in query predicates and index it. Not a problem, just struck me.

Reply:
I believe we're running out of `varchar` space for `5.7`. I may be mistaken but I have this recollection from a few months ago when I tried adding a `varchar` column.

Reply:
Ah, we're probably hitting the [on] page size limit (text can be moved off page to secondary storage).

This is a good example of why we should probably start leveraging JSON columns more, otherwise we'll end up with hundreds of columns which may or may not be used at different times. We're using them now in VDiff2 FWIW.